shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -1408,15 +1408,19 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
         final String source;
         @ToString.Exclude
         final String delegationToken;
-        final List<ConditionalAttributeUpdate> attributeUpdates;
```

Comment:
This is not a backward compatible change in the wire protocol. See https://github.com/pravega/pravega/wiki/PDP-21-(Protocol-Revisioning)